webpack.config.js

```diff
@@ -27,7 +27,7 @@ module.exports = {
   module: {
     loaders: [{
       test: /\.js$/,
-      loaders: ['react-hot', 'babel'],
+      loaders: ['react-hot', 'babel?stage=0'],
```

Comment:
That is strange - because it does not work if I don't write it into the webpack file - any idea why?